distributed/deploy/cluster.py

```diff
@@ -248,8 +247,8 @@ def get_logs(self, cluster=True, scheduler=True, workers=True):
             self._get_logs, cluster=cluster, scheduler=scheduler, workers=workers
         )
 
+    @_deprecated(use_instead="get_logs")
```

Comment:
This changes the warning that's emitted from a `DeprecationWarning` to a `FutureWarning`. This is intentional as `Cluster.logs` is a user-facing method we want users to move away from.